### PR TITLE
Add minikube-ci/examples to docs

### PR DESCRIPTION
**Pull request by medyagh**

### CODE REVIEW

**Comment by spowelljr** [comment]
Screen Shot 2021-08-23 at 4 00 42 PM
I'm assuming these are supposed emojis?

**Comment by sharifelgamal** [comment]
theoretically those emoji codes should work in markdown, but in my experience, just copy and pasting the emoji directly into the source works much better.

also the Gitlab link is broken.

**Comment by medyagh** [comment]
> theoretically those emoji codes should work in markdown, but in my experience, just copy and pasting the emoji directly into the source works much better.
> 
> also the Gitlab link is broken.

nice catch ! thanks it should be fixed now

File: site/content/en/docs/tutorials/continuous_integration.md [hunk 1]
```diff
@@ -2,14 +2,35 @@
...
 Most continuous integration environments are already running inside a VM, and may not support nested virtualization.
+You could use either `none` or `docker` driver in CI.
+
+To see a working example of running minikube in CI checkout [minikube-ci/examples](https://github.com/minikube-ci/examples) that contains working examples.
+
+
+## Supported / Tested CI Platforms
+
+
+For any platform not yet listed we are looking for your help! Please file Pull Requests and / or Issues for missing CI platforms :smile:
```
Comment (sharifelgamal) [inline site/content/en/docs/tutorials/continuous_integration.md:20]: `:smile:` should be replaced with 😄

File: site/content/en/docs/tutorials/continuous_integration.md [hunk 2]
```diff
@@ -2,14 +2,35 @@
...
+| [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) | [Yes](https://github.com/minikube-ci/examples/blob/master/azure-pipelines.yml) ✔️ | [![Azure Pipelines](https://dev.azure.com/medyagh0825/minikube-ci/_apis/build/status/examples?api-version=5.1-preview.1)](https://dev.azure.com/medyagh0825/minikube-ci/_build) 
+| [Travis CI](https://travis-ci.com/) | [Yes](https://github.com/minikube-ci/examples/blob/master/.travis.yml) ✔️ | [![Travis CI](https://travis-ci.com/minikube-ci/examples.svg?branch=master)](https://travis-ci.com/minikube-ci/examples/) |
+| [CircleCI](https://circleci.com/) | [Yes](https://github.com/minikube-ci/examples/blob/master/.circleci) ✔️ | [![CircleCI](https://circleci.com/gh/minikube-ci/examples.svg?style=svg)](https://circleci.com/gh/minikube-ci/examples) |
+| [Gitlab](https://about.gitlab.com/product/continuous-integration/) | [Yes](https://github.com/minikube-ci/examples/blob/master/.gitlab-ci.yml) ✔️ | ![Gitlab](https://gitlab.com/minikube-ci/examples/badges/master/pipeline.svg) |
```
Comment (sharifelgamal) [inline site/content/en/docs/tutorials/continuous_integration.md:30]: the issue with this url is that you need to be signed in to see it, so having it embedded just leaves it broken

File: site/content/en/docs/tutorials/continuous_integration.md [hunk 3]
```diff
@@ -2,14 +2,35 @@
...
+
+## Supported / Tested CI Platforms
+
+
+For any platform not yet listed we are looking for your help! Please file Pull Requests and / or Issues for missing CI platforms :smile:
+
+| Platform | Known to Work? | Status |
+|---|---|--|
+| [Prow](https://github.com/kubernetes/test-infra/tree/master/prow) | [Yes](https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes/minikube) ✔️ | [![Prow](https://prow.k8s.io/badge.svg?jobs=pull-minikube-build)](https://prow.k8s.io/?job=pull-minikube-build) |
+| [Google Cloud Build](https://cloud.google.com/cloud-build/) | [Yes](https://github.com/minikube-ci/examples/blob/master/gcb.md) :heavy_check_mark: | [![cloud build status](https://storage.googleapis.com/minikube-ci-example/build/working.svg)](https://pantheon.corp.google.com/cloud-build/dashboard?project=k8s-minikube) |
```
Comment (sharifelgamal) [inline site/content/en/docs/tutorials/continuous_integration.md:25]: replace `:heavy_check_mark:` with ✔️

**Review by sharifelgamal — CHANGES_REQUESTED** [review]

File: site/content/en/docs/tutorials/continuous_integration.md (hunk 2 above)
Comment (medyagh) [reply site/content/en/docs/tutorials/continuous_integration.md:30]: good call

**Review by sharifelgamal — APPROVED** [review]

**Review by spowelljr — APPROVED** [review]

**Pull request merged**